sinch/domains/verification/models/responses.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+from sinch.core.models.base_model import SinchBaseModel
+
+
+@dataclass
+class StartVerificationResponse(SinchBaseModel):
```

Comment:
This seems to be tied to a response for a Start Verification with `sms` method
What about other Start responses (`flashCall`, `callout` and `data`) ?

Reply:
fixed

Reply:
see Antoine's comment about dedicated classes

Reply:
done 👌

Reply:
Nice. I think you can also update the dataclass below as `StartFlashCallInitiateVerificationResponse` declares only the element `flash_call` but now it declares more elements in the `handle_response` method

Reply:
```
@dataclass
class StartFlashCallInitiateVerificationResponse(StartVerificationResponse):
    flash_call: dict
```
StartFlashCallInitiateVerificationResponse gets rest of its properties from inheritance - StartVerificationResponse.